packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -217,6 +217,9 @@ export default class Proxy {
       }),
       'vm-data/xoa-updater-channel': JSON.stringify(xoProxyConf.channel),
     }
+    if (httpProxy !== undefined) {
+      xenstoreData['vm-data/xoa-updater-proxy-url'] = JSON.stringify(httpProxy)
```

Comment:
Do you confirm that it expects a `JSON.stringify`ed string here and not just the string?

Reply:
Yes, this entry will be parsed in the updater side.